File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_disk_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusDiskVO:
    """文件存储盘信息VO"""
    created_at: str = None
    updated_at: str = None
    id: int = None
    uuid: str = None
    name: str = None
    owner: str = None
    owner_id: int = None
    disk_size: int = None
    used_size: int = None
    description: str = None
    remaining_size: int = None
    usage_rate: float = None

File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusDiskVO.kt
package com.sdkwork.backend

data class PlusDiskVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val id: Int? = null,
    val uuid: String? = null,
    val name: String? = null,
    val owner: String? = null,
    val ownerId: Int? = null,
    val diskSize: Int? = null,
    val usedSize: Int? = null,
    val description: String? = null,
    val remainingSize: Int? = null,
    val usageRate: Double? = null
)
